docs/reference/analysis/analyzers/whitespace-analyzer.asciidoc

```diff
@@ -4,3 +4,8 @@
 An analyzer of type `whitespace` that is built using a
 <<analysis-whitespace-tokenizer,Whitespace
 Tokenizer>>.
+
+The whitespace analyzer splits ((("whitespace analyzer")))the text on whitespace. It doesn't
+lowercase. It would produce
++
+    Set, the, shape, to, semi-transparent, by, calling, set_trans(5)
```

Comment:
This example is from the Def Guide, but without any context...  Would be helpful to expand the example